### PR TITLE
Add 95th and 99th percentile to timer metrics

### PR DESCRIPTION
**Pull request by tomasv**
95th and 99th percentile are very useful to have. Right now we only get the 50th and 90th through statsd.

@vLukas-dev @buinauskas @davidkazlauskas

### CODE REVIEW

File: src/metric_processor.rs [hunk 1]
```diff
@@ -106,6 +110,10 @@ mod test {
                      buckets.timer_data().get("some.timer.stddev").unwrap());
         assert_float("23.400",
                      buckets.timer_data().get("some.timer.upper_90").unwrap());
+        assert_float("23.400",
```
Comment (tomasv) [inline src/metric_processor.rs]: The values are the same because there's only 4 metric values in buckets. Should I add more?

Comment (buinauskas) [reply src/metric_processor.rs]: That would add slightly more confidence, wouldn't it?

Comment (tomasv) [reply src/metric_processor.rs]: A little. But it means that for me to get a unique 99th I need to generate a list of 100 numbers, which seems overkill.

Comment (buinauskas) [reply src/metric_processor.rs]: That's true as well. Given that the intention is to move to prometheus, I'd say I'm ok. Besides that, you're not writing code to calculate percentiles. So in an essence you'd be testing code that you didn't write.

**Review by buinauskas — APPROVED** [review]

**Comment by tomasv** [comment]
Added more metrics to get unique 95th ant 99th values in test.

**Review by lukas-kairevicius — APPROVED** [review]

**Pull request merged**